app/src/store/cart.ts

```diff
@@ -393,7 +393,8 @@ export default function useCartStore() {
     return cart.value
       .map((item) =>
         (grantRounds.value || []).reduce((inRound: boolean, round) => {
-          return inRound || grantRoundMetadata.value[metadataId(round.metaPtr)].grants?.includes(item.grantId) || false;
+          const metadata = grantRoundMetadata.value[metadataId(round.metaPtr)];
+          return inRound || metadata?.grants?.includes(item.grantId) || false;
```

Comment:
Sanity check—the fix here is the added `?` before accessing the `grants` key right?

Reply:
And this means #577 was not actually related to a user adding "Grants which are both in a round and not in a round to their cart and then refreshes the page"? i.e. it's seems like this just has to do with refreshing on the cart page?

Reply:
Thats correct, I wasn't exactly sure what was going on when I wrote the issue up 😅